diff --git a/dist/index.js b/dist/index.js
index 024fbe26f4b46af7c1a9d88022127086c43abec3..3d90ef5a3c091c214647077f1949fa9c1246bedc 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,4 +1,4 @@
-import { remarkParseSuperSubScript, } from './remark-plugins/parse-super-sub-script';
+import { remarkParseSuperSubScript, } from './remark-plugins/parse-super-sub-script.js';
 export default function rspressPluginSupersub(options = {}) {
     return {
         name: 'rspress-plugin-supersub',
