diff --git a/dist/index.js b/dist/index.js
index 4c8b235b6a7429c552ec41755da7a7270949f780..c8f7ec148291a38c9501e1772b73bd1b09ab62eb 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,3 +1,4 @@
+import { createRequire } from 'node:module';
 import remarkMath from 'remark-math';
 import rehypeKatex from 'rehype-katex';
 import { visit } from 'unist-util-visit';
@@ -16,7 +17,8 @@ const remarkCodeBlockToMath = () => {
     };
 };
 export default function rspressPluginKatex(options = {}) {
-    const katexCss = require.resolve('katex/dist/katex.min.css');
+    const _require = createRequire(import.meta.url);
+    const katexCss = _require.resolve('katex/dist/katex.min.css');
     return {
         name: 'rspress-plugin-katex',
         // config not needed for Rspress V2
